# Description: Set a minimum viewport width for sites that don't specify one
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/third_party/WebKit/Source/core/css/resolver/ViewportStyleResolver.cpp b/third_party/WebKit/Source/core/css/resolver/ViewportStyleResolver.cpp
--- a/third_party/WebKit/Source/core/css/resolver/ViewportStyleResolver.cpp
+++ b/third_party/WebKit/Source/core/css/resolver/ViewportStyleResolver.cpp
@@ -55,20 +55,18 @@ ViewportStyleResolver::ViewportStyleReso
 }
 
 void ViewportStyleResolver::collectViewportRules()
 {
     CSSDefaultStyleSheets& defaultStyleSheets = CSSDefaultStyleSheets::instance();
     collectViewportRules(defaultStyleSheets.defaultStyle(), UserAgentOrigin);
 
     bool useMobileViewportStyle = m_document->settings() && m_document->settings()->useMobileViewportStyle();
-#if OS(ANDROID)
     // FIXME: Remove when useMobileViewportStyle is set from chromium.
     useMobileViewportStyle = true;
-#endif
     if (useMobileViewportStyle)
         collectViewportRules(defaultStyleSheets.defaultMobileViewportStyle(), UserAgentOrigin);
 
     if (m_document->isMobileDocument())
         collectViewportRules(defaultStyleSheets.defaultXHTMLMobileProfileStyle(), UserAgentOrigin);
 
     if (ScopedStyleResolver* scopedResolver = m_document->scopedStyleResolver())
         scopedResolver->collectViewportRulesTo(this);
